src/pytest_html/scripts/main.js

```diff
@@ -41,10 +26,10 @@ const renderStatic = () => {
 }
 
 const renderContent = (tests) => {
-    const renderSet = tests.filter(({ when }) => when === 'call')
+    const renderSet = tests.filter(({ when, outcome }) => when === 'call' || outcome === "Error" )
```

Comment:
out of scope for this pr, but I would like to create constants for outcome and when, so we do this:
`outcome === ERROR` instead of `outcome === 'Error'`

Reply:
Yes, I like that!